cmd/key_rotate.go

```diff
@@ -81,13 +81,18 @@ func BuildCmdRotateKey(w io.Writer, handler RotateKeyHandler, rf *RootFlags) *co
 		"",
 		"Path to the file containing the wallet's passphrase",
 	)
-	cmd.Flags().StringVarP(&f.PublicKey,
-		"pubkey", "k",
+	cmd.Flags().StringVarP(&f.NewPublicKey,
+		"new-pubkey", "",
 		"",
 		"A public key to rotate to. Should be generated by wallet's 'generate' command.",
 	)
+	cmd.Flags().StringVarP(&f.CurrentPubKey,
+		"current-pubkey", "",
+		"",
+		"A public key to rotate from. Should be currently used public key.",
+	)
 	cmd.Flags().Uint64VarP(&f.TxBlockHeight,
-		"tx-height", "x",
+		"tx-height", "",
```

Comment:
Since we removed it from the the `tx-height` we should be consistent and remove it from the `target-height` as well. Sorry for not catching it earlier.